### ISSUE TITLE
Docker Generated python-client incorrect imports

### ISSUE THREAD

**Issue by dhdavvie** [esi/esi-issues#760]
# Bug
I followed the instructions [here](https://developers.eveonline.com/blog/article/swagger-codegen-update) to download the python-client folder using Docker. I then followed the following steps to setup the environment:  
```
$ virtualenv -p python3 venv
$ source venv/bin/activate
$ pip install -r requirements
```
With this setup, I get this error when importing `swagger_client` :  
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/__init__.py", line 19, in <module>
    from swagger_client.api.alliance_api import AllianceApi
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/api/__init__.py", line 6, in <module>
    from swagger_client.api.alliance_api import AllianceApi
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/api/alliance_api.py", line 21, in <module>
    from swagger_client.api_client import ApiClient
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/api_client.py", line 27, in <module>
    import swagger_client.models
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/models/__init__.py", line 194, in <module>
    from swagger_client.models.get_fw_leaderboards_characters_ok import GetFwLeaderboardsCharactersOk
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/models/get_fw_leaderboards_characters_ok.py", line 20, in <module>
    from swagger_client.models.get_fw_leaderboards_characters_victory_points import GetFwLeaderboardsCharactersVictoryPoints  # noqa: F401,E501
  File "/Users/dh/Documents/Projects/Eve/python-client/swagger_client/models/get_fw_leaderboards_characters_victory_points.py", line 19, in <module>
    from swagger_client.models.get_fw_leaderboards_characters_active_total1 import GetFwLeaderboardsCharactersActiveTotal1  # noqa: F401,E501
ModuleNotFoundError: No module named 'swagger_client.models.get_fw_leaderboards_characters_active_total1'
```

Looking at the files, `get_fw_leaderboards_characters_active_total1` seems to have been changed to `get_fw_leaderboards_characters_active_total_1` and the import functions were not updated. I fixed it locally by simply finding all the instances where these similar imports were and changing them manually. The affected files were:  
- get_fw_leaderboards_characters_victory_points.py
- get_fw_leaderboards_active_total.py
- get_fw_leaderboards_corporations_victory_points.py
- get_fw_leaderboards_victory_points.py

**Comment by hk135** [comment]
I am getting the same issue with the client generated from swagger.io, started replacement but the issue goes further than get_fw_leaderboards_characters_active_total1 with other files having the same issue.

**Comment by a-tal** [comment]
I have a fix for this in staging, to enforce unique model titles. the only affected routes are in the fw endpoints

<img width="631" alt="screen shot 2018-05-17 at 4 02 11 pm" src="https://user-images.githubusercontent.com/1832772/40207952-e3da1e80-59eb-11e8-8b11-3f58f4588345.png">

**Comment by a-tal** [comment]
thanks, this is resolved as of `ESI 0.8.2`

**Comment by alansalinas** [comment]
I have the same problem trying to use the generated python client from https://editor.swagger.io/ using ESI 0.8.3
